egs2/qasr_tts/conf/tuning/finetune_joint_conformer_fastspeech2_hifigan.yaml

```diff
@@ -0,0 +1,271 @@
+# This EXPERIMENTAL configuration is for ESPnet2 to finetune
```

Comment:
For initial PR, it would be better to make the config concise. Could you try to remove some redundant config for the start?

Reply:
Yes sure